config/default/test/targets/deaths.spec.js

```diff
@@ -0,0 +1,74 @@
+const chai = require('chai');
+const expect = chai.expect;
+const path = require('path');
+const moment = require('moment')
+const sinon = require('sinon');
+const TestRunner = require('medic-conf-test-harness');
+const { babyDeceasedAtAge1Day, babyDeceasedAtAgeJustUnder5Years, babyDeceasedAtAge5Years } = require('../contacts');
+const harness = new TestRunner({
+  xformFolderPath: path.join(__dirname, '../../forms/app'),
+});
+
+let clock;
+
+describe('Death related targets tests', () => {
+  before(async () => {
+    return await harness.start();
+  });
+  after(async () => {
+    return await harness.stop();
+  });
+  beforeEach(async () => {
```

Comment:
I think you should just be able to have `beforeEach(harness.clear)` for this. And some of these other one-line functions here in the harness's boilerplate.

Reply:
I tried it but got this error:
```
"before each" hook: clear for "death this month target test this month":
     TypeError: Cannot read property 'inputs' of undefined
      at Context.clear (node_modules\medic-conf-test-harness\src\harness.js:116:39)
```